### ISSUE TITLE
Reopening #9294 error in RNN tutorial

### ISSUE THREAD

**Issue by numericlee** [tensorflow/tensorflow#18405]
Thanks to @kevinashaw for pointing out [various](https://github.com/tensorflow/tensorflow/issues/9294) problems with the RNN tensorflow tutorial

Although the issue was closed, as far as I can see @MarkDaoust @drpngx @ebrevdo @martinwicke 
the tutorial code posted here and at TF site still has the same errors after nearly 12 months

I did try making some of the modifications suggested in the thread, they don't appear to solve the issue

Have I written custom code: NO

OS Platform and Distribution: Windows 10; 
Python v. 3.5 
TensorFlow installed from: Anaconda
TensorFlow version 1.2.0
GPU model and memory: CPU 16GB of RAM
Exact command to reproduce (see below)
```
Xin =tf.random_normal((100,100))

batch_size_var  = tf.shape(Xin)[0]
lstm = tf.nn.rnn_cell.LSTMCell(lstm_size)
hidden_state = lstm_cells.zero_state(batch_size_var, tf.float32) 
current_state = lstm_cells.zero_state(batch_size_var, tf.float32)
```

There was a suggestion to use  dynamic rnn . It wasnt fully explained but using  dynamic rnn renders the reference to .LSTMCell  invalid

Is there another RNN tutorial or a revised version anyone can recommend?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by numericlee** [comment]
done

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 30 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 45 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 60 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 75 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 90 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 105 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 59 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 74 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 89 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 104 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 119 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by jvishnuvardhan** [comment]
@numericlee This is a stale issue. Since this issue opened, there were lot of tutorials and guides created on TF and keras websites. A few of the tutorials are here [Text_generation](https://www.tensorflow.org/text/tutorials/text_generation), [RNN guide](https://www.tensorflow.org/guide/keras/rnn), [text classification](https://www.tensorflow.org/text/tutorials/text_classification_rnn), [Time Series](https://www.tensorflow.org/tutorials/structured_data/time_series).

I am closing this issue as this was resolved. Thanks!
